### PR TITLE
Finished task2

### PR DESCRIPTION
**Pull request by Bazilchuk2020**

### CODE REVIEW

**Comment by andrii-s-spitzer-lpnu-ua** [comment]
you need get grade for task #1 first. After that you can do task#2

File: ConsoleApp18/ConsoleApp18/bin/Debug/net7.0/ConsoleApp18.dll
Comment (andrii-s-spitzer-lpnu-ua) [inline ConsoleApp18/ConsoleApp18/bin/Debug/net7.0/ConsoleApp18.dll:1]: Please remove all binary files. Git can not manage binary files.

**Review by andrii-s-spitzer-lpnu-ua — CHANGES_REQUESTED** [review]

File: ConsoleApp18/ConsoleApp18/bin/Debug/net7.0/ConsoleApp18.dll
Comment (Bazilchuk2020) [reply ConsoleApp18/ConsoleApp18/bin/Debug/net7.0/ConsoleApp18.dll:1]: I cannot see inary files, can you write name of this files and i remove it

**Comment by andrii-s-spitzer-lpnu-ua** [comment]
ConsoleApp18/ConsoleApp18/bin/Debug/net7.0/ConsoleApp18.exe
ConsoleApp18/ConsoleApp18/bin/Debug/net7.0/ConsoleApp18.dll
and all other dll and exe files...

**Review by andrii-s-spitzer-lpnu-ua — APPROVED** [review]

**Pull request merged**